flavors/lineageos/default.nix

```diff
@@ -49,11 +49,11 @@ let
   filterDirsAttrs = dirs: mapAttrs (n: v: filterDirAttrs v) dirs;
 in mkIf (config.flavor == "lineageos")
 {
-  androidVersion = mkDefault 10;
+  androidVersion = mkDefault 11;
```

Comment:
I actually mean that we would set the default `androidVersion` here based on what device the user had chosen, and using information from `lineage-build-targets` which should end up in `device-metadata.json`.